web/lang/ru_ru.php

```diff
@@ -59,7 +58,7 @@
 // is subtlely different in versions of PHP before and after 4.3.0, see
 // http://uk2.php.net/manual/en/function.setlocale.php for details.
 // Also be aware that changing the whole locale may affect some floating point or decimal
-// arithmetic in the database, if this is the case change only the individual locale areas
+// arithmetic in the database, if this is the case change only the individual locale Зонаs
```

Comment:
Find and replace is working too well, like this and including swapping "areas" into Russian in the middle of untranslated phrases.